Makefile.local

```diff
@@ -1,24 +1,145 @@
-# This is the local Makefile for the projectcalico/api repository, with targets
-# specific to github.com/projectcalico/api. This is opposed to Makefile, which
+# This is the local Makefile for the tigera/api repository, with targets
+# specific to github.com/tigera/api. This is opposed to Makefile, which
 # is mirrored from github.com/projectcalico/calico/api.
 
+SHELL = /bin/bash
+
+# Local variables
+
+# The revision of the local repository; this is used as a marker for where/when
+# the PR was generated
+LOCAL_REPOSITORY_REVISION ?= $(shell git describe --tags)
```

Comment:
This is a good point, and this information was more useful in a previous design. I've removed it and replaced it with `$(CALICO_VERSION)` (so that it's easier to see the relevant version in the pull requests page), e.g. `Automatic API update from tigera/calico-private v3.20`.